doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
+      * Branches in the nodejs/node repository are only for release lines
     * [See "Updating Node.js from Upstream"](./onboarding-extras.md#updating-nodejs-from-upstream)
-    * make new branches for all commits you make!
+    * Make new branches for all commits you make!
```

Comment:
[question to a native english speakers]
`raise` vs `create` vs `request`
Intuitively would use `raise` for Issue, `request` for PR (since PR = "Pull Request", but turned into a noun)? or is `create` best?